src/test/Testcafe/designer_tests/ux-react-app-tests.js

```diff
@@ -0,0 +1,17 @@
+import { Selector, t } from 'testcafe';
+import App from '../app';
+import Page from '../page-objects/page';
+
+app = new App();
+page = new Page();
+
+
+
+fixture('UX react app tests')
+    .page(app.basepage)
+    .beforeEach()
+
+test('Repeating groups', () => {
+    await t
```

Comment:
The lamda-function must be async, so it will `await t.Click();`, right?

Reply:
fixed non async lambda function